rubygem/spec/assets/exit.rb

```diff
@@ -1 +0,0 @@
-exit(1)
```

Comment:
I got rid of this test case since I stopped catching `Exception` in `track_features_loaded_by`. I think someone would expect `exit` to work pretty promptly without waiting to pass a bunch of messages around.